wsgi/odyssy/announcement/models.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.core.exceptions import ValidationError
+
+# Create your models here.
+class Announcement(models.Model):
+    """ Model for Announcement
+    key         -- Primary Key
+    initDate    -- Initialization Date
+    finDate     -- Final Date
+    title       -- Title of the Announcement
+    description -- Description of the Announcement
+    """
+
+    def __str__(self):
+        return "#" + str(self.key) + " " + self.title
+
+    def save(self, force_insert=False, force_update=False, using=None,
```

Comment:
This method is fine but there are some drawbacks. See [2nd answer by stefanw here](https://stackoverflow.com/questions/2281179/adding-extra-constraints-into-fields-in-django).